generator/cli.py

```diff
@@ -0,0 +1,474 @@
+#!/usr/local/bin/python
```

Comment:
you need to add an empty `__init__.py` file to the folder (I think for the tests as well) in order to allow python to discover the modules and tests correctly.